src/mongooseFiles/routes/cities.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const { Router } = require('express');
+const { getRandomItem } = require('../../helpers');
+const Cities = require('../models/city');
+
+const router = Router();
+
+// GET
+router.get('/', (req, res) => {
+  Cities.find({}, (error, cities) => {
+    if (error) {
+      console.log(error);
+    } else {
+      res.send(cities);
+    }
+  });
+});
+
+// GET random city
+router.get('/randomCity', (req, res) => {
+  Cities.find({}, (error, cities) => {
+    if (error) {
+      console.log(error);
+    } else {
+      res.send(getRandomItem(cities));
+    }
+  });
+});
+
+// POST
+router.post('/', (req, res) => {
+  const city = new Cities();
```

Comment:
Modifying existing objects is bad practice, so in your case it's better to create object with already filled fields. Also it's good practice to destructurize your arguments - like with interface segregation principle, you will take only what you need and not single byte of data more than needed.

```javascript
router.post('/', ({
  body: {
    country,
    name,
    capital,
    location_lat,
    location_lng,
  }
}, res) => {
  const city = new City({
    country,
    name,
    capital,
    location: {
      lat: location_lat, //maybe it's better to send them camelcased?
      lng: location_lng,
    }
  });
  city.save((error, city) => {
    if (error) console.log(error);
    res.send(city);
  });
});
```

such way is safest way possible and it would work no matter of what library you use. however, mongoose doesn't save values from your object, if they doesn't present  in `strict` scemas. So, if you'll change you frontend representation from sending you `location_lat` and `location_lng` to sending an object `location: {lat, long}` you'll be able just to provide request body without any changes on backend:
```javascript
router.post('/', ({ body }, res) => new City(body)
  .save((error, city) => {
    if (error) console.log(error);
    res.send(city);
  })
);
```
or even change it to one-liner with async wrapper :)